### DISCUSSION TITLE
is there a way to trust the docker official image via signedby?

### DISCUSSION THREAD

**Discussion by braindevices** [containers/podman#15219] [Q&A]
`docker trust inspect` shows:
```
Signatures for alpine:latest

SIGNED TAG   DIGEST                                                             SIGNERS
latest       7580ece7963bfa863801466c0a488f11c86f85d9988051a9f9c68cb27f6b7872   (Repo Admin)

Administrative keys for alpine:latest

  Repository Key:       5a46c9aaa82ff150bb7305a2d17d0c521c2d784246807b2dc611f436a69041fd
  Root Key:     a2489bcac7a79aa67b19b96c4a3bf0c675ffdf00c6d2fabe1a5df1115e80adce
```

I wonder which key should we use in `podman trust` command? Or is this even possible?

**Comment by mheon** [comment]
The Docker signature scheme and the ones used by Podman are, I believe, incompatible, so I do not believe we can use these keys; @mtrmac can you confirm?

**Comment by rhatdan** [comment]
Correct Podman does not work with docker notary. Podman supports simple signing and soon sigstore.
